src/AddTodoForm.js

```diff
@@ -24,7 +23,7 @@ const AddTodoForm = ({ onAddTodo }) => { // Accept onAddTodo as a prop
           name="title"
           required
           value={todoTitle}
-          onChange={handleChange}
+          onChange={(event) => setTodoTitle(event.target.value)}
```

Comment:
Nice work deviating from the curriculum and writing a one-liner. To make it even shorter it's worth mentioning that `event` can be anything, and a common pattern is to just use `e`. No need to change it, especially if it makes it less readable for you, just wanted to show you since you are more experimental than the other folks I review.

```suggestion
          onChange={(e) => setTodoTitle(e.target.value)}
```